src/containers/AccountConnection.jsx

```diff
@@ -120,15 +121,15 @@ class AccountConnection extends Component {
       })
   }
 
-  updateAccount (connector, account, values) {
+  updateAccount (connector, account, values, successTimeout) {
```

Comment:
I suggest to remove `successTimeout` as parameter and use instead
```js
const { successTimeout } = this.state
```
in the method's body.